pkg/tfbridge/walk/walk.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2016-2023, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package walk
+
+import (
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim/walk"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
+)
+
+type SchemaPath = walk.SchemaPath
```

Comment:
I don't understand when we use the distinction between a `walk.SchemaPath` and a `SchemaPath`. This library uses both.

Reply:
It's an alias exact same thing.

Reply:
Well the key idea is that code outside this package does to need to import the other walk package just to reference SchemaPath. I can tidy up.